### DISCUSSION TITLE
How to connect to the database services in Codespaces using the GUI database management software

### DISCUSSION THREAD

**Discussion by myesn** [community/community#18772] [Codespaces]
Hi, thank you very much for the free Codepsaces service provided by GitHub for individual users, it's really awesome!

Here's what happened: I started a `PostgreSQL` database service in Codepsaces using Docker Compose with `ports: "5432:5432"` as configured by Docker Compose, as shown in the following figure:

![image](https://user-images.githubusercontent.com/18598579/174212978-b8e87af1-28eb-489a-a4f5-34ccb989db02.png)

Then I tried to connect to the database service started in Codespaces using the `DBeaver` database management software, but the connection was never successful, as shown in the following image:

*I am very sorry, the software I use here is in Chinese language, you may not understand it.*

![image](https://user-images.githubusercontent.com/18598579/174214078-6c8a2e80-4540-4bc3-9c1c-2d1462a72323.png)

**Comment by drachehavoc** [comment]
did you solve this problem?
